main/telemetry.js

```diff
@@ -11,7 +11,9 @@ const Sentry = require('@sentry/node')
 const client = new InfluxDB({
   url: 'https://eu-central-1-1.aws.cloud2.influxdata.com',
   // station-anonymous-write
-  token: '0fZyu9zjDvYlaNfOeuwgnQoUI0VcSzeYDpnOLjQyr30mz-Plqels5JHEwgKRbtCcDJbQmv62VnOV_FsZVxgoow=='
+  token:
+    '0fZyu9zjDvYlaNfOeuwgnQoUI0VcSzeYDpnOLjQyr30mz-Plqels5JHEwgKRbtCcDJbQmv62' +
+      'VnOV_FsZVxgoow=='
```

Comment:
Uh oh, splitting a magic string like this into two lines will make this code difficult to change when we rotate the token.

Let's ignore the max-len rule on this line please.